src/cq_cam/routers.py

```diff
@@ -211,23 +213,24 @@ def route_wires(job: "Job", wires: List[Union[cq.Wire, cq.Edge]], stepover=None)
         else:
             distance, target, param = None, None, None
 
-        if (
```

Comment:
The first original if is supposed to check if the next position is below the last position and then just do a straight plunge down (instead of a rapid_to which would lift all the way up before coming down again. 

Although judging from test_profile.test_profile_stepdown I don't think it's even working.